### ISSUE TITLE
labelTextColor

### ISSUE THREAD

**Issue by danniing** [n4kz/react-native-material-dropdown#110]
Hi,
It would be nice to have labelTextColor prop.
Is there any way to edit the color of the label without editing the underline?

I tried using baseColor but it changes the color of a lot more than just the label.

I also tried adding color to labelTextStyle but it doesn't seem to do anything.

Thanks for this Component!

**Comment by harshaSenaratne** [comment]
yeah i have the same issue regarding the label color, it would be nice if someone could help !

**Comment by ProfessionalAmateur** [comment]
I also need this feature.

**Comment by ravina26unikwork** [comment]
we can pass "color" props directly to change label text color also,
    <Dropdown
        ...
        color={Colors.white}
    />
